lib/gui/components/safe-webview.js

```diff
@@ -46,6 +46,22 @@ const VERSION_PARAM = 'etcher-version';
 const ELECTRON_SESSION = 'persist:success-banner';
 
 /**
+ * @summary API version search parameter key
+ * @constant
+ * @private
+ * @type {String}
+ */
+const API_VERSION_PARAM = 'api-version';
+
+/**
+ * @summary Webview API version
+ * @constant
+ * @private
+ * @type {String}
+ */
+const API_VERSION = 1;
```

Comment:
@jviotti mentioned that we should just treat no given `API_VERSION` as such, which we can call the the "zeroth" API version I guess.